ci

```diff
@@ -14,7 +14,7 @@ case "${1:?}" in
                -v "$(pwd)":/lua-resty-jwt \
                -w /lua-resty-jwt \
                --name lua-resty-jwt-tests \
-               cdbattags/openresty-testsuite:latest \
+               vinayakhulawale/openresty-testsuite:latest \
```

Comment:
I'll prob edit this as part of the final merge to mine if that's ok with you

Reply:
yes, absolutely ! I have already submitted another PR to your repo - https://github.com/cdbattags/openresty-testsuite/pull/1/.  Please merge that PR and publish new docker image before changing it.

Reply:
done